src/SFA.DAS.AssessorService.Data/AssessmentOrgsImporter.cs

```diff
@@ -0,0 +1,185 @@
+                    _logger.LogError(message, ex);
+                    progressStatus.Append(message);
+                    throw;
+
+                }
+            }
+
+            return new AssessmentOrgsImportResponse { Status = progressStatus.ToString() };
+        }
+
+        private async Task<AssessmentOrganisationsSpreadsheetDto> HarvestSpreadsheetData(StringBuilder progressStatus)
+        {        
+            var spreadsheetDto = new AssessmentOrganisationsSpreadsheetDto();
+
+            var containerName = "assessmentorgs";
+            var initialiseContainer = new BlobContainerHelper(_configuration);
+
+            var container = initialiseContainer.GetContainer(containerName).Result;
+
+            var blob = container.GetBlockBlobReference(TemplateFile);
+
+            var memoryStream = new MemoryStream();
+            await blob.DownloadToStreamAsync(memoryStream);
+
+            using (var package = new ExcelPackage(memoryStream))
+            {
+                progressStatus.Append("Reading from spreadsheet: Delivery Areas; ");
+                spreadsheetDto.DeliveryAreas = _spreadsheetReader.HarvestDeliveryAreas();
+                progressStatus.Append("Reading from spreadsheet: Organisation Types; ");
+                spreadsheetDto.OrganisationTypes = _spreadsheetReader.HarvestOrganisationTypes();
+                progressStatus.Append("Reading from spreadsheet: Organisations; ");
+                spreadsheetDto.Organisations =
+                    _spreadsheetReader.HarvestEpaOrganisations(package, spreadsheetDto.OrganisationTypes);
+                progressStatus.Append("Reading from spreadsheet: Standards; ");
+                var standards = _spreadsheetReader.HarvestStandards(package);
+                progressStatus.Append("Reading from spreadsheet: Organisation-Standards; ");
+                spreadsheetDto.OrganisationStandards =
+                    _spreadsheetReader.HarvestEpaOrganisationStandards(package, spreadsheetDto.Organisations,
+                        standards);
+                progressStatus.Append("Reading from spreadsheet: Organisation-Standards-Delivery Areas; ");
+                spreadsheetDto.OrganisationStandardDeliveryAreas =
+                    _spreadsheetReader.HarvestStandardDeliveryAreas(package, spreadsheetDto.Organisations, standards,
+                        spreadsheetDto.DeliveryAreas);
+                progressStatus.Append("Reading from spreadsheet: Contacts; ");
+                spreadsheetDto.Contacts = _spreadsheetReader.HarvestOrganisationContacts(spreadsheetDto.Organisations,
+                    spreadsheetDto.OrganisationStandards);
+            }
+            return spreadsheetDto;
+
+        }
+
+        private void TearDownDatabase(StringBuilder progressStatus)
+        {
+            _logger.LogInformation($"Teardown process started; ");
+            var progress = _assessmentOrgsRepository.TearDownData();
+            progressStatus.Append((string)progress);
```

Comment:
Bit picky here. I am assuming this is already a string then remove the redundant cast.